### PR TITLE
Fixing Typing Errors

### PR DESCRIPTION
**Pull request by passabilities**
It fixes errors caused by TypeScript.

### CODE REVIEW

File: src/common/utxobased/db/makeProcessor.ts [hunk 1]
```diff
@@ -119,31 +119,42 @@ export async function makeProcessor(config: ProcessorConfig): Promise<Processor>
...
     createOrOpen(disklet, utxoIdsBySizeConfig)
   ])
+  const hashBases = Promise.all([
+    createOrOpen(disklet, addressByScriptPubkeyConfig),
+    createOrOpen(disklet, txByIdConfig),
+    createOrOpen(disklet, txsByScriptPubkeyConfig),
+    createOrOpen(disklet, utxoByIdConfig),
+    createOrOpen(disklet, utxoIdsByScriptPubkeyConfig)
+  ])
+  await Promise.all([countBases, rangeBases, hashBases])
```
Comment (thehobbit85) [inline src/common/utxobased/db/makeProcessor.ts:139]: I understand you have issues with the types but doing the "await Promise.all" in this line and also doing individually "await" on "countBases/rangeBases/hashBases" in the next lines is redundent.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
Fix the "Promise.all" issue

File: src/common/utxobased/db/makeProcessor.ts (hunk 1 above)
Comment (passabilities) [reply src/common/utxobased/db/makeProcessor.ts:139]: I don't have an issue with it, TypeScript throws errors when the array for the promise all is being destructed like this with more than 10 items:

image

Comment (thehobbit85) [reply src/common/utxobased/db/makeProcessor.ts:139]: I understand that. You need to do something like this:

```
const [counts,  ranges , hashs ] = await Promise.all([countBases, rangeBases, hashBases])
const [
    scriptPubkeyByPath,
    addressPathByMRU
  ] = counts
  const [
    scriptPubkeysByBalance,
    txIdsByBlockHeight,
    txsByDate,
    utxoIdsBySize
  ] = await ranges
  const [
    addressByScriptPubkey,
    txById,
    txsByScriptPubkey,
    utxoById,
    utxoIdsByScriptPubkey
  ] = await hashs
```

Comment (passabilities) [reply src/common/utxobased/db/makeProcessor.ts:139]: I mean that is the same thing I'm doing without creating additional variables

**Pull request merged**